index.js

```diff
@@ -771,6 +774,11 @@
     return x instanceof Right && R.eqProps('value', x, this);
   });
 
+  //  Right#extend :: Either a b ~> (b -> b) -> Either a b
+  Right.prototype.extend = def('Right#extend', [Function], function(f) {
+    return Right(f(this.value));
```

Comment:
Absolutely. The documentation states that:
```
Though this is a constructor function the `new` keyword needn't be used.
```
so I think we should be consistent in that regard.